app/models/concerns/tiny_mce_images.rb

```diff
@@ -79,6 +79,49 @@ def clone_tinymce_assets(target, team)
       target.reassign_tiny_mce_image_references(cloned_img_ids)
     end
 
+    def copy_unknown_tiny_mce_images
+      asset_team_id = case self.class.name
+                      when 'Protocol'
+                        team_id
+                      when 'Step' || 'MyModule'
+                        protocol.team_id
+                      when 'ResultText'
+                        result.my_module.protocol.team_id
+                      end
+
```

Comment:
Can this be extracted into separate method? I guess it could come handy in other situations as well.

Reply:
Moved team id searching to Team model